src/index.js

```diff
@@ -7,6 +7,10 @@ const supportMultiple = (typeof document !== 'undefined' && document && document
   'multiple' in document.createElement('input') :
   true;
 
+const walkDirectory = Symbol('walk directory');
```

Comment:
This was carried over from the BetterView coding style. Since symbols are unique, the are used in this manner to mimic private methods.